src/main/java/gregtech/api/unification/ore/OrePrefix.java

```diff
@@ -531,6 +536,23 @@ public void processOreRegistration(@Nullable Material material) {
         }
         if (material != null) {
             generatedMaterials.add(material);
+            if (material.isHidden() && material.hasFluid()) {
```

Comment:
This also needs to check for a potential plasma. It only checks for non-plasma fluids right now, so a mod registering a plasma with the same name as CEu can still have problems. Additionally, materials can have plasmas without fluids.